js/everything.js

```diff
@@ -435,21 +435,26 @@ function anchorTimestamps(text, videoID) {
       mm:ss
       m:ss
     and wraps the timestamps in <a> tags
+    RegEx explanation:
+    ((?:[0-5]\d|\d|) either the string is "colon 00-59" or "0-9" or "blank"
+    (?:\d|\:[0-5]\d) either the string is "colon 0-9" or "colon 00-59"
+    (?:$|\:[0-5]\d)) either the string ends or is a a number between 00-59
     */
     var re = /((?:[0-5]\d|\d|)(?:\d|\:[0-5]\d)(?:$|\:[0-5]\d))/g;
-    return text.replace(re, (match, $1) => "<a href=\"" + makeListenURL(videoID, convertTimestamp($1)) + "\">" + $1 + "</a>");
+    return text.replace(re, function(match) { return "<a href=\"" + makeListenURL(videoID, convertTimestamp(match)) + "\">" + match + "</a>"; });
```

Comment:
Can we do something like this? I'd rather not use a function in `.replace()` since it's a bit clunky

```javascript
text.replace(re, "<a href=\"" + makeListenURL(videoID, convertTimestamp("$1")) + "\">$1</a>");
```